README.md

```diff
@@ -8,20 +8,61 @@ You can install GitLab Community Edition for testing purposes.
 
 First, install **Docker**.
 
+Second, create a docker volume for persistent data : `docker volume create vol-gitlab`.
+
 Then, you can simply run :
 
 ```
 docker run --detach \
 	--hostname gitlab.example.com \
 	--publish 443:443 --publish 80:80 --publish 22:22 \
 	--name gitlab \
+	--mount source=vol-gitlab,target=/app \
 	--restart always \
 	--volume /srv/gitlab/config:/etc/gitlab \
 	--volume /srv/gitlab/logs:/var/log/gitlab \
 	--volume /srv/gitlab/data:/var/opt/gitlab \
 	gitlab/gitlab-ce:latest
 ```
 
-Or, for Windows PowerShell : `docker run --detach --hostname gitlab.example.com --publish 443:443 --publish 80:80 --publish 22:22 --name gitlab --restart always --volume /srv/gitlab/config:/etc/gitlab --volume /srv/gitlab/logs:/var/log/gitlab --volume /srv/gitlab/data:/var/opt/gitlab gitlab/gitlab-ce:latest`
+Or, for Windows PowerShell : `docker run --detach --hostname gitlab.example.com --publish 443:443 --publish 80:80 --publish 22:22 --name gitlab --mount source=vol-gitlab,target=/app --restart always --volume /srv/gitlab/config:/etc/gitlab --volume /srv/gitlab/logs:/var/log/gitlab --volume /srv/gitlab/data:/var/opt/gitlab gitlab/gitlab-ce:latest`
 
 > **Note :** do not use Windows PowerShell ISE
+
+### Customisable Rules
+
+You can customise the existing rules, by adding yours. Rules are quiet simply to extend.
+They are in this `rules.yml` file, located in `src/rules/rules.yml` and must respect the `Rule` class (`src/rules/rules.ts`).
```

Comment:
Can we override the built-in rules ? 
For our ones, we should maybe add a concrete example.